src/hello.rs

```diff
@@ -231,6 +231,9 @@ pub(crate) struct HelloCommandResponse {
 
     /// For internal use.
     pub topology_version: Option<Document>,
+
+    /// The maximum permitted size of a BSON wire protocol message.
+    pub max_message_size_bytes: Option<i32>,
```

Comment:
is this field ever actually not present in a hello response? or could it be non-optional here and in `StreamDescription`?

Reply:
Looks like it's always there, updated.